CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate Data JDBC Client
 Unreleased
 ==========
 
+ - The Crate JDBC driver loads automatically now.
```

Comment:
Suggestion: "Applications no longer need to explictly load Crate JDBC using Class.forName()"